cdap-e2e-tests/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -3,3 +3,21 @@ clientUrl=http://localhost:11011
 serverUrl=https://placeholder.com/api
 # command to generate token: gcloud auth print-access-token
 serverAccessToken=placeholder
+
+## NAMESPACEADMIN-PROPERTIES-START
+nameSpacePreferences1=[{"key":"dataset","value":"test_automation"}]
+nameSpacePreferences2=[{"key":"projectId","value":"cdf-athena"},{"key":"datasetprojectId","value":"cdf-athena"}]
+validProfile=TestProfile
+validDescription=TestDescription
+validClusterName=TestClusterName
+invalidNamespaceName=^%&&3%%
+validNamespaceDescription=Test Description
+invalidProfile=6*&gjh879
+invalidClusterName=$^%%&^GHJJH89
+validNamespaceName=AutoTestt
+ConnectionName=TestConnection
+projectId=cdf-athena
+dataset=test_automation
+#bqSourceTable=dummy
+#bqTargetTable=dummy
```

Comment:
Please remove the commented parameter values if not used.

Reply:
Removed